unit-tests/heitech.zer0mqXt.core.tests/PubSubTests.cs

```diff
@@ -74,6 +72,29 @@ public async Task Subscriber_Cancellation_works()
             
         }
 
+        [Fact]
+        public async Task PubSub_with_retry_works()
+        {
+            // Arrange
+            Message capturedResponse = null;
+            using var socket = ConfigurationTestData.BuildInProcSocketInstanceForTest("retry-socket", timeoutInMs: 500, usePblshr: true);
+            var waitHandle = new ManualResetEvent(false);
+            // request in background thread
+            socket.RegisterSubscriber<Message>(msg => { capturedResponse = msg; waitHandle.Set();});
+            // wait a second for retry
```

Comment:
Comment seems incorrect.

Reply:
yes, copy pasta. It needs to read: setup subscriber for handling in background thread.

Reply:
done